examples/__init__.py

```diff
@@ -29,7 +29,7 @@
     from .tf.graphsage.graph_sage import GraphSage
     from .tf.line.line import LINE
     from .tf.transe.trans_e import TransE
-except ImportError:
-    pass
+except ImportError as e:
+    print(e)
```

Comment:
Don't do that, just failed sliently and let user know it when they do use those examples.